python/lib/python_port.c

```diff
@@ -36,9 +36,22 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "python_port.h"
 #include "reactor.h"
+#include "api/api.h"
+#include "api/set.h"
 
 PyTypeObject py_port_capsule_t;
 
+//////////// destructor Function(s) /////////////
+/**
+ * Called when putting token to the recycling bin. This function decreases the 
+ * reference count of PyObject and allows python to collect garbage later.
+ * @param py_object A PyObject with count 1*
```

Comment:
To conform with our coding style, comment should be something like this;
```suggestion
 * Decrease the reference count of PyObject. When the reference count hits zero,
 * Python can free its memory.
 * @param py_object A PyObject with count 1 or greater.
```
We use the imperative tense for comments, and the comment should describe what the function does, not where it's called. (I'm not sure what you meant by count `1*`).